gc/stats/AllocationStats.hpp

```diff
@@ -37,6 +37,7 @@ class MM_AllocationStats : public MM_Base
 	uintptr_t _tlhRefreshCountFresh; /**< Number of refreshes where fresh memory was allocated. */
 	uintptr_t _tlhRefreshCountReused; /**< Number of refreshes where TLHs were reused. */
 	uintptr_t _tlhAllocatedFresh; /**< The amount of memory allocated fresh out of the heap. */
+	uintptr_t _tlhAllocatedUsed; /**< The amount of memory has been flushed */
```

Comment:
to align comment with the variable name: amount of used memory of already flushed TLHs